addons/portal_rating/static/src/js/portal_composer.js

```diff
@@ -74,6 +74,22 @@ PortalComposer.include({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * Returns a non-resolving Promise (to prevent multiple submissions) after
+     * enforcing a non-empty textarea to ensure rating is submitted.
+     *
+     * @override
+     * @private
+     * @returns {Promise}
+     */
+    _onSubmitButtonClick: function () {
+        var $textarea = this.$('textarea');
+        if (!$textarea.val()) {
+            $textarea.val(' ');
+        }
+        return this._super.apply(this, arguments);
```

Comment:
I think it has come to the point where we will not even bother doing that.
We're trying to "fix" an edge case (user not filling up the comment) with a half solution here.

We may want to re-discuss this whole thing with SBU and decide exactly what we want to to, and implement it in a clean way.
If the comment is sometimes required, sometimes not, and we want to keep it that way, we will try to find a way to make it a parameter somewhere in the code, instead of filling up the comments with blank strings.

Sorry that you have lost some time with this, but that's what comes with R&D, we have to try and guess and sometimes we end up not doing anything at all.

Reply:
Understood, I will drop the change then. It was a good learning experience though! Also sorry about all the comments/comment changes, I didn't realize you get a notification every time.

Reply:
Don't worry about it.

Good tip though, if you have a review where you have a lot of comments to answer to, you can go in the "Files changed" view and "start a review" when writing your answers.

That way all your answers are part of a review and when you submit it when you're done, the reviewer only gets one notification.